website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/example-customizations.mdx

```diff
@@ -7,190 +7,153 @@ description: Example customizations for the ServiceNow Service Catalog Integrati
-      1. Open the "Script step" within the Action. Remove any example variables
-         and add your custom variables by clicking "Create Variable" at the
+      1. Add your custom variables by clicking the **Create Input** button. Ensure that the variable names match your Catalog Item variables and select the variable type that matches each variable. Click **Save**.
+      ![Screenshot: ServiceNow Add Variables To Action](/img/docs/servicenow-catalog-add-variables-to-action.png "Screenshot of adding ServiceNow Action Input Variables")
+      1. Open the **Script step** within the Action. Remove any example variables
+         and add your custom variables by clicking **Create Variable** at the
          bottom. Enter the name of each variable and drag the corresponding data
-         pill from the right into the "Value field".
+         pill from the right into the **Value field**.
          ![Screenshot: ServiceNow Add Script Step Variables To Action](/img/docs/servicenow-catalog-adjust-script-variables.png "Screenshot of adjusting ServiceNow Action Script Variables")
-      1. Click "Save" and then "Publish" to save the changes made to the Action.
-1. Finally, reopen the Flow and attach the newly created Action to the Flow
+      1. Click **Save** and then **Publish**.
+1. Reopen the Flow and attach the newly created Action to the Flow
    after "Get Catalog Variables" step:
    1. Remove the previous Action that you copied earlier and replace it with
       your newly created Action.
       ![Screenshot: ServiceNow Replace Action Step](/img/docs/servicenow-catalog-replace-action.png "Screenshot of replacing ServiceNow Action step")
    1. Connect the new Action to the Flow by dragging and dropping the data pills
       from the "Get Catalog Variables" Action to the corresponding inputs of
-      your new Action. Click "Done" to save this step.
+      your new Action. Click **Done** to save this step.
       ![Screenshot: ServiceNow Fill Variables for Action](/img/docs/servicenow-catalog-fill-new-action-step.png "Screenshot of filling out ServiceNow Action variables")
-   1. Once the Flow is properly configured, click "Save" at the top to save the
-      changes.
-   1. Click "Activate" to enable your Flow and make it available for use.
-
-### Set the Flow for your Catalog Item
-
-1. Navigate back to the Catalog by clicking on "All" and then go to "Service
-   Catalog > Catalogs" followed by "Terraform Catalog".
-1. Locate your custom Catalog Item and click on the three dots icon at the top
-   of the item. From the dropdown menu, select "Configure item".
-1. In the configuration settings, click on the "Process Engine" tab.
-1. In the "Flow" field, search for the Flow you recently created. Once you find
-   it, click on it to select it and then click the "Update" button to save the
-   changes.
+   1. Click **Save**.
+   1. Click **Activate**.
```

Comment:
Now that you removed the brief context around these steps, it's not clear what's the difference between Save and Activate.